app/src/main/res/values/dimens.xml

```diff
@@ -74,4 +74,11 @@
     <dimen name="row_header_width">55dp</dimen>
     <dimen name="text_size">14sp</dimen>
     <dimen name="table_header_line">1dp</dimen>
+    <dimen name="text_size_35sp">35sp</dimen>
```

Comment:
why are we using new padding? What's the issue with the padding used previously?

Reply:
It is the text size of the 'Mifos Mobile' logo at the top. If you want a different text size then I can change it.

Reply:
can you check the `dimen` below, why are we using new padding?

Reply:
these paddings are used in redesigning of `activity_login.xml`